modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -17,6 +17,69 @@
 toc::[]
 
 
+== Infinite Scale 7.2.0 (Production) - Addams 
+
+Please find the most important changes described here or refer to the changelog for a complete list of changes:
+
+* Infinite Scale: {ocis-releases-url}/v7.2.0[Changes in 7.2.0, window=_blank]
+* Web: {web-releases-url}/v12.0.3[Changelog for ownCloud Web 12.0.3, window=_blank]
+
+[discrete]
+=== Summary of Notable Feature Implementations
+
+* You can now manage Spaces via the IDP by using the `OCIS_CLAIM_MANAGED_SPACES_ENABLED` setting. Once enabled, space management will no longer be possible via the web UI, only the IDP, and it will be part of the claim. All services using this environment variable must be configured. See the xref:next@ocis:ROOT:{s-path}/deployment/services/env-vars-special-scope.adoc[Environment Variables with Special Scopes] for more details.
```

Comment:
```suggestion
* You can now manage Spaces via the IDP by using the `OCIS_CLAIM_MANAGED_SPACES_ENABLED` setting. Once enabled, space management will no longer be possible via the web UI, only the IDP, and it will be part of the claim. All services using this environment variable must be configured. See xref:next@ocis:ROOT:{s-path}/deployment/services/env-vars-special-scope.adoc[Environment Variables with Special Scopes] for more details.
```